test/utilities/utilities.go

```diff
@@ -0,0 +1,271 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package utilities
```

Comment:
nit

```suggestion
package util
```

Reply:
Although I was happy with `framework` too, to remain consistent with kubernetes/kubernetes

Reply:
Yea I mainly decided to leave framework as is because the setup differs a lot between integration and e2e.  And we were already using framework to setup the envtest.  So I figured these were really test utilites and the frameworks could be used for the envtest setup.  

But I'm open to change on that.

Reply:
sounds good. Please rebase